src/themes/dspace/styles/_global-styles.scss

```diff
@@ -17,7 +17,7 @@
     background-color: var(--bs-primary);
   }
 
-  h4 {
+    h4 {
```

Comment:
@GauravD2t : Can you remove the space added here?  We want to avoid reformatting code unless it's necessary as it makes the PR very messy to review.

Reply:
remove the space befor <h4> tag